screenshots/url_formulas.py

```diff
@@ -0,0 +1,31 @@
+from datetime import date 
+from datetime import timedelta
+
+# NOTE: the %#d (day of month not zero padded) and %#m (number of month not zero padded) formatting directives 
+#       are different depending on whether you're running on unix or windows. Unix is %-d, windows is %#d. 
+# NOTE: %B (full month name) makes capitalized months, which doesn't match the existing urls. The web server was doing an extra redirect to sort it out, so I did tolower().
+
+# MA PRIMARY
+# =CONCATENATE("https://www.mass.gov/doc/covid-19-dashboard-",LOWER(TEXT(now(),"mmmm")),"-",LOWER(TEXT(now(),"d")),"-" , YEAR(NOW()),"/download")
+print("https://www.mass.gov/doc/covid-19-dashboard-december-9-2020/download")
+print(eval("date.today().strftime(\"https://www.mass.gov/doc/covid-19-dashboard-%B-%#d-%Y/download\").lower()"))
+
+# MA SECONDARY
+# =CONCATENATE("https://www.mass.gov/doc/weekly-covid-19-public-health-report-",LOWER(TEXT(now(),"mmmm")),"-",LOWER(TEXT(now(),"d")),"-" , YEAR(NOW()),"/download")
+print("https://www.mass.gov/doc/weekly-covid-19-public-health-report-december-9-2020/download")
+print(eval("date.today().strftime(\"https://www.mass.gov/doc/weekly-covid-19-public-health-report-%B-%#d-%Y/download\").lower()"))
+
+# MT TERTIARY
+# =CONCATENATE("https://dphhs.mt.gov/Portals/85/publichealth/documents/CDEpi/DiseasesAtoZ/2019-nCoV/Status%20Update%20Data%20Report_",MONTH(NOW()),if(DAY(NOW())<10,CONCATENATE(0,DAY(NOW())),DAY(NOW())),"20",TEXT(NOW(),"y"),".pdf")
```

Comment:
Would this accept partial day offsets? I updated this URL yesterday based on the time that MT seems to update this PDF and the current URL builder looks like:
`=CONCATENATE("https://dphhs.mt.gov/Portals/85/publichealth/documents/CDEpi/DiseasesAtoZ/2019-nCoV/Status%20Update%20Data%20Report_",MONTH(NOW()-0.9),if(DAY(NOW()-0.9)<10,CONCATENATE(0,DAY(NOW()-0.9)),DAY(NOW()-0.9)),"20",TEXT(NOW()-0.9,"y"),".pdf")`
That said, while this might seem to be the most accurate, it is super ugly. -1 would probably do the trick.